lbrynet/lbryfilemanager/EncryptedFileDownloader.py

```diff
@@ -59,10 +60,13 @@ def restore(self):
         self.claim_id = yield self.wallet.get_claimid(self.uri, self.txid, self.nout)
         status = yield self.lbry_file_manager.get_lbry_file_status(self)
         if status == ManagedEncryptedFileDownloader.STATUS_RUNNING:
+            log.info("lbry://%s (%s) is running", self.uri, self.sd_hash[:6])
```

Comment:
Please don't repeat the `self.sd_hash[:6]` part.  If you want that to be the standard way of logging an sd_hash, than make a function or anything that allows you to change how its logged if you so choose in the future.  It also ensures that you're consistent with how these sorts of things are logged.